openedx/features/subscriptions/models.py

```diff
@@ -0,0 +1,49 @@
+import logging
+
+from datetime import date
+from django.contrib.auth.models import User
+from django.db import models, transaction
+from model_utils.models import TimeStampedModel
+
+from student.models import CourseEnrollment
+
+class UserSubscription(TimeStampedModel):
+    """
+    The "UserSubscription" model.
+    """
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    subscription_id = models.PositiveIntegerField(db_index=True)
+    sku = models.CharField(
```

Comment:
In which case `sku` is null?

Reply:
It shouldn't be null in case of successful creation of a subscription product. However, the existing API for Course modes did it so I followed pattern.